Jenkinsfile.oscontainer

```diff
@@ -0,0 +1,23 @@
+def DOCKER_IMG = "registry.fedoraproject.org/fedora:28"
```

Comment:
Let's just standardize on `coreos-assembler`?

Reply:
This is just to run `sed` though right?

Reply:
Sure, I just like the idea of that being the default container for all pipelines.
(Did you see https://github.com/cgwalters/coreos-assembler/pull/1 BTW?).